src/main/java/com/force/i18n/grammar/impl/LanguageDeclensionFactory.java

```diff
@@ -270,6 +270,12 @@ private LanguageDeclension createDeclension(HumanLanguage language) {
             return new UnsupportedLanguageDeclension.HaitianCreoleDeclension(language);
         case GREENLANDIC:
             return new UnsupportedLanguageDeclension.GreenlandicDeclension(language);
+        case CHUJ:
+            return new ChujDeclension(language);
+        case KICHE:
+            return new KicheDeclension(language);  
+        case KAQCHIKEL:
+            return new KicheDeclension(language);            
```

Comment:
KAQCHIKEL uses KICHE Declension based on the [doc](https://salesforce.quip.com/AkiCA4CcitM3#temp:C:OFP1e28e27abeb44c6495b318625).